source/SIL.AppBuilder.Portal.Frontend/src/ui/components/project-table/table/row/products/index.tsx

```diff
@@ -11,24 +12,34 @@ type IProps =
 
 class Products extends React.Component<IProps> {
   render() {
-    const { activeProductColumns, t, products } = this.props;
+    const { activeProductColumns, activeProjectColumns, t, products } = this.props;
     const hasProducts = products && products.length > 0;
 
+    let padding = activeProjectColumns.length - activeProductColumns.length;
```

Comment:
I guess we can extract this code into a util?

Reply:
unsure, it's only used in one other place, and is kind of specific to this scenario, so I'm not sure if it would be used anywhere else